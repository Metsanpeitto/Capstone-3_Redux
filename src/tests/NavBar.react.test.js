import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Link } from 'react-router-dom';
import Next from '../imgs/next.svg';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router basename={process.env.PUBLIC_URL}>
        <Link to="/" active="true" exact="true">
          <img src={Next} alt="return" className="arrow-left" />
        </Link>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
