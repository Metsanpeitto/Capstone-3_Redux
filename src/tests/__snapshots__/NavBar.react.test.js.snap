// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  active="true"
  exact="true"
  href="/"
  onClick={[Function]}
>
  <img
    alt="return"
    className="arrow-left"
    src="next.svg"
  />
</a>
`;
